src/DSharp.Mscorlib/DateTime.cs

```diff
@@ -22,11 +22,11 @@ public sealed class DateTime
         [ScriptAlias("ss.DateTime.Equals")]
         public extern static bool Equals(DateTime d1, DateTime d2);
 
-        public extern DateTime(int year, int month, int day);
+        public DateTime(int year, int month, int day) { }
```

Comment:
We can make this class a struct btw :)
Also how come we removed the externs here?

Reply:
extern constructors generate warnings, so it's better to leave them empty

Reply:
I made it a `readonly struct`